railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -395,7 +395,6 @@ def delete_action_mailer_files_skipping_action_mailer
 
       def delete_action_cable_files_skipping_action_cable
         if options[:skip_action_cable]
-          remove_file "config/cable.yml"
           remove_file "app/assets/javascripts/cable.js"
           remove_dir "app/channels"
```

Comment:
Why are the other files created then? Shouldn't they all be skipped before this or deleted here in one go?

Having two separate mechanisms seems weird.